packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -51,9 +51,9 @@ void main() {
     // to be part of the word sometimes and not others, which is fine, but we'd mildly prefer if
     // we were consistently considering them part of words always.
     final TextRange hebrew1 = painter.getWordBoundary(const TextPosition(offset: 4, affinity: TextAffinity.downstream));
-    expect(hebrew1, const TextRange(start: 0, end: 8), skip: skipExpectsWithKnownBugs);
+    expect(hebrew1, const TextRange(start: 0, end: 8), skip: skipExpectsWithKnownBugs); // https://github.com/flutter/flutter/issues/87536
```

Comment:
Should this annotation just be on the `skipExpectsWithKnownBugs` constant?

Reply:
That would make it harder for the analyzer script to detect it. Right now I am think it would just use a simple regexp looking for the skip param with a comment on the same line.